owfs.html

```diff
@@ -61,8 +62,8 @@
         category: 'function',
         defaults: {
             name: {value:""},
-            host: {value:"localhost",required:true},
-            port: {value:4304,required:true,validate:RED.validators.number()},
```

Comment:
It seems that the numbers validator does not allow an empty field, which I was a bit surprised about, so I replaced it with a regex which does allow that.  In fact the regex is arguably better anyway as number would presumably allow a decimal point.